### PR TITLE
Update README.md to add megablocks requirement for mixtral

### PR DESCRIPTION
**Pull request by 0-hero**
Update README.md to add megablocks requirement for mixtral

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +72,10 @@ Install vLLM with pip or [from source](https://vllm.readthedocs.io/en/latest/get
 ```bash
 pip install vllm
 ```
+Mixtral required additional requirement `megablocks` which can be installed with pip or [from source](https://github.com/stanford-futuredata/megablocks):
```
Comment (WoosukKwon) [inline README.md:75]:
```suggestion
**NOTE:** The Mixtral model additionally requires `megablocks` which can be installed with pip or [from source](https://github.com/stanford-futuredata/megablocks) on **Python 3.10**:
```

**Review by WoosukKwon — CHANGES_REQUESTED** [review]
Hi @0-hero Thanks for the PR! Left a minor suggestion. Please check it out.

**Comment by 0-hero** [comment]
Updated

**Review by WoosukKwon — APPROVED** [review]
LGTM! Thanks for helping the documentation!

**Pull request merged**